README.md

```diff
@@ -1 +1,34 @@
-# women-in-open-source
+# Women in Open Source
+
+Description
+___________________
+
+## Code of Conduct
+
+Please refer to the Hacktoberfest FFM Code of Conduct, which can be found [here](https://hacktoberffm.de/conduct).
+
+_______________
+
+## Template
+
+This template should be used when adding your name to the list. Please copy-paste the template and then change the information accordingly.
+
+* Name
+  * [Twitter handle](https://www.twitter.com/)
+  * Personal website/ portfolio page/ Github
+  * Interests: (e.g. programming language, technology)
```

Comment:
Interests should probably be topics/projects, because we are adding persons for their activities in the Open Source ecosystem 😊